hydradx/tests/test_money_market.py

```diff
@@ -765,13 +764,13 @@ def test_set_mm_oracles_to_external_market():
                 ltv=0.6
             )
             for tkn in omnipool.asset_list
-        ]
+        ], oracle_source=omnipool.usd_price
     )
-    prices = [{tkn: omnipool.price(tkn, "USDT") for tkn in omnipool.asset_list}]
-    state = GlobalState(pools={"omnipool": omnipool}, money_market=mm, agents={}, external_market=prices[0])
-    money_market_update(prices)(state)
+    prices = {tkn: omnipool.price(tkn, "USDT") for tkn in omnipool.asset_list}
+    mm.oracle_source = omnipool.usd_price
```

Comment:
This was probably me, but there's no reason we should be setting the oracle source to the same thing twice.

Reply:
Fixed